build/gulpfile.sql.js

```diff
@@ -145,3 +149,20 @@ gulp.task('package-rebuild-extensions', task.series(
 	task.define('clean-rebuild-extensions', () => ext.cleanRebuildExtensions('.build/extensions')),
 	task.define('rebuild-extensions-build', () => ext.packageRebuildExtensionsStream().pipe(gulp.dest('.build'))),
 ));
+
+
+gulp.task('external-translations-import', function () {
```

Comment:
Add description of what this task is doing please

Reply:
Where is this being used? If it's not relevant to the issue being fixed then for future situations like this it would be great to split it out into separate PRs or at least clearly indicate that it's not directly related.

Reply:
This is for importing the i18n resources into the extension, and it also generates the localized package.json file for extensions

Reply:
Please add that as a comment in the code for future people looking at this 😄

Reply:
And it would be good to be extremely clear about what is actually happening. What does it mean to "import the i18n resources into the extension"?

Reply:
For each extension and each language, It copies the localized json files of the i18n/<language>/extensions/<extension name> folder into an i18n folder inside the actual extension folder. It also creates a package.nls.<language>.json file in the extension folder's base directory, to localize the package.json file.

The end result will be an i18n folder inside the extension that has language folders inside containing localized json files, and there will be several package.nls.<language>.json files in the base directory of the extension.